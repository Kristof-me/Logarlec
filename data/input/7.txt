create room r1
create student s1 r1
create student s2 r1
effect wet r1
effect gas r1
split r1 r2 d12
status r1 actors
status r2 actors
status effects r1
status effects r2