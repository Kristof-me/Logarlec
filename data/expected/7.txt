s1
s2
-
víz: 10
gáz: 10
víz: 10
gáz: 10
